static/js/components/EducationForm.js

```diff
@@ -45,10 +46,10 @@ import type {
 import { formatMonthDate } from '../util/date';
 import FIELDS_OF_STUDY from '../data/fields_of_study';
 
-const fieldOfStudyOptions = _.map(FIELDS_OF_STUDY, (name, code) => ({
+const fieldOfStudyOptions = sortOptions(_.map(FIELDS_OF_STUDY, (name, code) => ({
   value: code,
   label: name
-}));
+})));
```

Comment:
There's a function called `labelSort` in `lib/currency.js` which I think you can re-use (unless I'm missing something).